packages/application-shell/src/components/navbar/navbar.js

```diff
@@ -130,28 +134,48 @@ MenuExpander.propTypes = {
   onClick: PropTypes.func.isRequired,
 };
 
-export const MenuGroup = props => (
-  <ul
-    className={classnames(
-      { [styles.list]: props.level === 1 },
-      { [styles.sublist]: props.level === 2 },
-      {
-        [styles['sublist-no-children']]: props.level === 2 && !props.children,
-      },
-      {
-        [styles['sublist-expanded__active']]:
-          props.level === 2 && props.isActive && props.isExpanded,
-      },
-      props.level === 2 && props.isActive && !props.isExpanded
-        ? styles['sublist-collapsed__active']
-        : styles.sublist__inactive
-    )}
-  >
-    {props.children}
-  </ul>
-);
+export const MenuGroup = props => {
+  const isSublistActiveWhileIsMenuExpanded =
+    props.level === 2 && props.isActive && props.isExpanded;
+  const isSublistActiveWhileIsMenuCollapsed =
+    props.level === 2 && props.isActive && !props.isExpanded;
```

Comment:
Nice, the rest just moved I suppose.

Reply:
Yes, I extracted those into meaningful variables, as I'm using them for the `aria-expanded` attribute